api/data_overview_builder.py

```diff
@@ -0,0 +1,125 @@
+"""
+Data Overiew Builder generates a markdown file with information about data in the API.
+
+By default this is shown on the docs site.
+
```

Comment:
how and when is this run? How does someone running it check and deploy the output?

Reply:
This is run when you run `./run.py api update-schemas`.  You can check the output by running the docs locally and it's deployed through the normal docs deploy process `tools/deploy-docs.sh` (I do realize this is not obvious or really stated anywhere, will try to add to readme

Reply:
Thank you for adding to the README :-)